app/build.gradle.kts

```diff
@@ -73,6 +73,7 @@ android {
 }
 
 dependencies {
+    implementation(project(":domain:camera"))
```

Comment:
nit: move this to the bottom, similar to the other build.gradle.kts files for internal project deps